### ISSUE TITLE
minimum size of an rbd image

### ISSUE THREAD

**Issue by pohly** [ceph/ceph-csi#85]
When I created a small test PVC with `1Mi` as size, that was rounded down to size 0:

```
I1010 09:49:43.820983       1 rbd_util.go:124] rbd: create pvc-c46136c5-cc71-11e8-b494-deadbeef0100 size 0G format 2 (features: layering) using mon 192.168.7.2:6789,192.168.7.4:6789,192.168.7.6:6789,192.168.7.8:6789, pool rbd id admin key AQAe07xbq/TsOhAAkLrH6vaRDo/xxVZg/Y6qJg==
```

```
# rbd info pvc-c46136c5-cc71-11e8-b494-deadbeef0100
rbd image 'pvc-c46136c5-cc71-11e8-b494-deadbeef0100':
	size 0 B in 0 objects
	order 22 (4 MiB objects)
	id: 5e36643c9869
	block_name_prefix: rbd_data.5e36643c9869
	format: 2
	features: layering
	op_features: 
	flags: 
	create_timestamp: Wed Oct 10 09:49:43 2018
```

Obviously the resulting `/dev/rbd0` then was unusable ;-}

Is there a minimum size?

**Comment by rootfs** [comment]
this is a bug here https://github.com/ceph/ceph-csi/blob/master/pkg/rbd/controllerserver.go#L100

PR welcome!

**Comment by lelvisl** [comment]
fixed in #213

**Comment by Madhu-1** [comment]
This is fixed as  stated in #213
